docs/index.html

```diff
@@ -2,69 +2,191 @@
 layout: default
 title: eBay Skin
 ds: 6
+ds_map:
+    alert-dialog:
+        name: alert-dialog
+        version: 2.1
+    avatar:
+        name: avatar
+        version: 1.0
+    badge:
+        name: badge
+        version: 1.2
+    breadcrumbs:
+        name: breadcrumb
+        version: 1.2
+    button:
+        group: button
+        name: button
+        version: 1.6
+    carousel:
+        name: carousel
+        version: 1.1
+    calendar:
+        name: calendar
+        version: 1.0
+    checkbox:
+        name: checkbox
+        version: 2.0
+    chip:
+        version: 1.0.0
+    chips-combobox:
+        name: chip-combobox
+        version: 1.0.0
+    combobox:
+        name: dropdown
+        version: 2.2
+    confirm-dialog:
+        name: alert-dialog
+        version: 2.1
+    cta-button:
+        group: button
+        name: button
+        version: 1.2
+    date-textbox:
+        name: date-picker
+        version: 1.0
+    details:
+        name: expansion
+        version: 1.2
+    drawer-dialog:
+        name: bottom-sheet
+        status: deprecated
+        version: 2.0
+    eek:
+        name: eek
+        version: 1.1
+    filter-button:
+        name: quick-filter
+        version: 2.0
+    filter-menu:
+        version: 2.0
+    filter-menu-button:
+        name: quick-filter
+        version: 2.0
+    flag:
+        version: 1.0
+    floating-label:
+        name: form-field
+        version: 2.2
+    fullscreen-dialog:
+        name: modal
+        version: 2.1
+    icon:
+        name: icon
+        version: 1.3
+    icon-button:
+        group: button
+        name: icon-button
+        version: 1.6
+    infotip:
+        name: infotip
+        version: 2.0
+    inline-notice:
+        name: alert-notice
+        version: 2.2
+    lightbox-dialog:
+        - name: modal
+          version: 2.1
+        - name: bottom-sheet
```

Comment:
Are these dependencies for DS because I'm not seeing `lightbox-dialog` in Skin having a bottom sheet or be dependent on it?

Reply:
Great question! These are the Design System components/patterns that Skin's `lightbox-dialog` component implements. In the Design System, `modal` and `bottom-sheet` are separate patterns, but in Skin there is a common implementation. So we list out both DS components here. Changing `name` to `ds-name` will help clarify; I'll implement that suggestion.

Reply:
Sounds good.